lib/Runtime/Base/ThreadContext.cpp

```diff
@@ -3030,7 +3031,10 @@ ThreadContext::ClearScriptContextCaches()
 {
     for (Js::ScriptContext *scriptContext = scriptContextList; scriptContext != nullptr; scriptContext = scriptContext->next)
     {
-        scriptContext->ClearScriptContextCaches();
+        if (!scriptContext->IsClosed())
```

Comment:
looks this is not necessary, I added it while tweaking the change. let me remove it